### PR TITLE
{bio}[GCC/11.2.0] VEP v105

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/11de65f6ab9bb5c7d912988952286142 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+checksums = ['12a6bc1f579513cac8b9167cce4e363655cc8eba26b7d9fe1170dfe95e044f42']
+
+builddependencies = [('pkg-config', '0.29.2')]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('BioPerl', '1.7.8'),
+    ('HTSlib', '1.14'),
+]
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: `preconfigopts = "env HTSLIB_DIR=$EBROOTHTSLIB"` is no longer necessary?

Comment (verdurin) [reply easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: @migueldiascosta could you say why? What has changed?

Comment (verdurin) [reply easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: @migueldiascosta could you say why? What has changed?

Comment (migueldiascosta) [reply easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: @verdurin I was asking a question :) but I see that it has been addressed in #15116, and this PR no longer needs to add this easyconfig?

Comment (boegel) [reply easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: Ah, no, #15116 also includes `Bio-DB-HTS-3.01-GCC-11.2.0.eb`, so there's a merge conflict here. I'll get that fixed then...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3513.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/f5e89766065ee6ba80c0138cb482ee3d for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**